core/pom.xml

```diff
@@ -188,5 +188,10 @@ Bundle-DocURL:
             <groupId>uk.org.lidalia</groupId>
             <artifactId>slf4j-test</artifactId>
         </dependency>
+		<!-- core component dependency added in core pom -->
+		<dependency>
+			<groupId>com.adobe.cq</groupId>
+			<artifactId>core.wcm.components.core</artifactId>
```

Comment:
@rakeshkumar-verma : you would need to add the version of  core components too. Also why is this change part of title PR ?

Reply:
removed.